prime-router/metadata/fhir_mapping/hl7/message/ORU_R01.yml

```diff
@@ -67,6 +67,7 @@ resources:
     additionalSegments:
       - .ORC
       - .NTE
+      - .TIMING_QTY.TQ1
```

Comment:
This one get me trouble.  These names are not just the segment names, they are what the parser names them.  I had to debug the library to find this answer.  Note the two layers too.

Reply:
I am not sure how to tackle this issue.  Understanding how the HAPI parser works was critical here, but I am unsure I can create or find a decoder ring for this.  I could document it for the segments we have, but that's it.